external/geogram/cmake/platforms/Linux64-clang/config.cmake

```diff
@@ -1,5 +1,9 @@
 set(VORPALINE_ARCH_64 true)
 include(${GEOGRAM_SOURCE_DIR}/cmake/platforms/Linux-clang.cmake)
-add_flags(CMAKE_CXX_FLAGS -m64)
-add_flags(CMAKE_C_FLAGS -m64)
 
+IF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
+    add_flags(CMAKE_CXX_FLAGS -m64)
+    add_flags(CMAKE_C_FLAGS -m64)
+ELSE(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
+    message("NOT ON ARM CPU! Omitting arm specific flag.")
+ENDIF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
```

Comment:
Is this omitting -m64 when not on ARM?

This worked before on x86 without omitting arm specific flags right. It seems that now when you are compiling on ARM, the behavior is the same as before, but on x86 there is no -m64/-m32. Do we need to change these?

Reply:
Yes, you are right and this is giving me a little bit of a headache.
I have tested this specifically only on the Khadas Vim3 pro board, which has arm Cortextes, but is running a custom Linux setup that is being developed by the manufacturer.
So I think on some levels/flags the Cmake will tell you that it is x86, but on others it will show arm. I think here it just does not match the `CMAKE_SYSTEM_PROCESSOR` with arm, because on that Linux system the processor doesn't have a specifier string, just a manufacturer- and type/generation-id.

TL;DR: this should actually be the other way around, because m32/64 are i386 specifics.
I think the most sensible solution for now is to switch around these if/else branches and add `NOT NATIVE_SSE` to them. 
What do you think @fwilliams

Reply:
With latest this compiles both on the arm test-board (with `NATIVE_SSE OFF`) and on my desktop Linux on x86.